include/proxy-wasm/wasm.h

```diff
@@ -42,11 +42,19 @@ using WasmForeignFunction =
 using WasmVmFactory = std::function<std::unique_ptr<WasmVm>()>;
 using CallOnThreadFunction = std::function<void(std::function<void()>)>;
 
+struct SanitizationConfig {
+  std::vector<std::string> argument_list;
+  enum class ListType : int { Allowlist = 0, Denylist = 1 };
+  ListType list_type;
```

Comment:
Do we need a new enum here? This is effectively `true`/`false`, so `bool` should be sufficient.

Reply:
Fixed.